index.js

```diff
@@ -139,38 +54,34 @@ export default class CountUp extends Component {
     startAnimation(this);
   }
 
-  spanElement = null;
-
-  refSpan = (span: Element<*>) => {
+  refSpan(span) {
     this.spanElement = span;
-  };
-
-  props: Props;
+  }
 
   render() {
-    const {
-      className,
-      start,
-      style,
-      decimal,
-      decimals,
-      useGrouping,
-      separator,
-      prefix,
-      suffix,
-    } = this.props;
-
     return (
-      <span className={className} style={style} ref={this.refSpan}>
-        {formatNumber(start, {
-          decimal,
-          decimals,
-          useGrouping,
-          separator,
-          prefix,
-          suffix,
+      <span
+        className={this.props.className}
```

Comment:
Can you pull these out as constants?
const { className, start, decimal, decimals } = this.props;